src/index.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './components/App';
+import rootReducer from './reducers/index';
+
+const reduxStore = createStore(rootReducer);
```

Comment:
**REQUIRED**
* It adds example books to initial state object. [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png).
See the [documentation](https://redux.js.org/api/createstore) on how to create store with a pre-loaded state using createStore.

Reply:
As per coding conventions and the majority of react-redux tutorials I have come across, the best-suited place for this initial state object is in the file you have the reducer as the main index.js file shouldn't contain inner core responsibilities or exports as it is not good practice. Please advise me further if anything
![Screenshot 2020-10-23 at 09 58 18](https://user-images.githubusercontent.com/57832450/96974529-a87c0d80-1519-11eb-9dba-37d1b60888d7.png)
.